Core/src/org/sleuthkit/autopsy/textextractors/TikaTextExtractor.java

```diff
@@ -158,7 +160,7 @@
                     .map(mt -> mt.getType() + "/" + mt.getSubtype())
                     .collect(Collectors.toList());
 
-    public TikaTextExtractor(Content content) {
+    TikaTextExtractor(Content content) {
```

Comment:
@wschaeferB @rcordovano I think this might technically be a public API change, since org.sleuthkit.autopsy.textextractors is a publicly exported package.

Reply:
My bad we @eugene7646, @rcordovano and I discussed something related but not the removal of the public keywords in these instances, that is a mistake as well caused by me for whatever reason thinking because there was no public constructor in a final class no one would be able to call any of those methods, I forgot that other classes in the package could still return TikaTextExtractor or a lookup could still find one. I will add the public keywords back and I am sorry for that mistake.

Reply:
@eugene7646 @rcordovano wait no the class itself isn't public but it had a public constructor, can that constructor ever even be used publicly if the class isn't public?